CryptoTechReminderSystem/DomainObject/TimeSheet.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using Newtonsoft.Json;
+
+namespace CryptoTechReminderSystem.DomainObject
+{
+    public class TimeSheet
+    {
+        [JsonProperty("id")] 
+        public int Id { get; set; }
+        [JsonProperty("spent_date")] 
+        public string SpentDate { get; set; }
```

Comment:
Might be worth renaming this? 

Rationale - How would I know what SpentDate refers to without looking at the Harvest API documentation?

Reply:
Strongly agree with Dushan.
I suggest `TimesheetDate` for the rename.

Reply:
Yes, the current domain object is pretty much identical to the harvest response, I think this will change depends on the use cases.

Reply:
Renamed for now, but I think the domain may change more in the future ab494d14e13e8346bd5d5a848b6849f2b6700347